pkg/util/storage.go

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
```

Comment:
Doesn't belong here - pkg/cloudprovider/storage/ is one option, or a sub package of util (storage)